tools/dashboard

```diff
@@ -7,6 +7,9 @@ cd "$(readlink -f "$(dirname "$0")")/.."
 
 if [ -n "${PROJECT}" -a "$1" != "auto" ]; then
   . config/options ""
+  BUILD_PREFIX=${BUILD_ROOT}/${BUILD_BASE}
```

Comment:
You don't need BUILD_PREFIX here - the purpose of sourcing `config/options` when a PROJECT is defined is to obtain THREAD_CONTROL. It's not doing any harm, you just won't follow a code path that ever uses BUILD_PREFIX when PROJECT is defined, so it's not required.

Reply:
dropped it in the new commit, but leave a comment for understanding.